src/Data/Functor/Day.purs

```diff
@@ -117,3 +124,9 @@ instance comonadDay :: (Comonad f, Comonad g) => Comonad (Day f g) where
 
 instance comonadTrans :: Comonad f => ComonadTrans (Day f) where
   lower = runDay \get fx gy -> get (extract fx) <$> gy
+
+instance foldableDay :: (Foldable f, Foldable g) => Foldable (Day f g) where
```

Comment:
This seems like an unusual choice of instance to me. That `length (day _ fx gy)` is `length fx * length gy` seems a bit strange. I know `Foldable` has no laws to satisfy, but still. I noticed this instance also isn't present in `kan-extensions`, so I'd be interested to hear if there's a reason why.

Reply:
My only real motivation for this was that it felt like it should have one, similar to how the Coyoneda instance works:
```purescript
toList (coyoneda t f) ≅ t <$> toList f
toList (day t f g) ≅ t <$> toList f <*> toList g
```

I'd also be interested to hear both: why it's not present in kan-extensions; and which alternative instances are possible.